kotlinpoet-classinspector-elements/src/main/kotlin/com/squareup/kotlinpoet/classinspector/elements/ElementsClassInspector.kt

```diff
@@ -201,10 +201,10 @@ class ElementsClassInspector private constructor(
     //       and compare against only preceding ones.
     val methodList = methodMap.asMap()[simpleName.toString()]?.toList()
         ?: return false
-    val indexOfPossibleOverrider = methodList.indexOf(this)
-    return (indexOfPossibleOverrider downTo 0)
-        .asSequence()
-        .map { methodList[it] }
+    val signature = jvmMethodSignature(types)
```

Comment:
Looks like the comment on line 198 is obsolete after this change. I'd vote for deleting it completely cause it tries to explain the algorithm that isn't complicated enough to warrant a comment.